src/Config_option.ml

```diff
@@ -84,20 +85,23 @@ module Make (C : CONFIG) = struct
       ?(allow_inline = Poly.(section = `Formatting)) ?(deprecated = false)
       update get_value =
     let open Cmdliner in
-    let invert_flag = default in
-    let names_for_cmdline =
-      if invert_flag then
-        List.filter_map names ~f:(fun n ->
-            if String.length n = 1 then None else Some ("no-" ^ n))
```

Comment:
I removed this check. This means that there is a `--no-q` flag to disable `--q`. Should I implement it back ?

Reply:
Yes I think we should keep it, `--no-q` looks weird IMO.

It was suggested in #299 to use `-Q` to negate `-q` for example, I'm not really a fan of this notation but if it makes sense to you it would be the right time to implement it.

Reply:
Did.